boilerplate/fullstack/astro/src/components/Home.tsx

```diff
@@ -32,7 +32,7 @@ export default function Home({
         if (hasInvalidClaims) {
             return <SessionAuthForAstro />;
         } else {
-            return <TryRefreshComponent />;
+            return <TryRefreshComponent key={Date.now()} />;
```

Comment:
might be good to add a comment as to why we are doing this.

Reply:
done